app/screens/edit_post/edit_post_input/edit_post_input.test.tsx

```diff
@@ -84,4 +86,15 @@ describe('EditPostInput', () => {
         const {queryByTestId} = renderWithEverything(<EditPostInput {...props}/>, {database, serverUrl});
         expect(queryByTestId('uploads')).toBeNull();
     });
+
+    it('should render quick actions in edit mode', () => {
```

Comment:
0/5 another test case to make sure the actions don't appear when it is not edit mode?

And what about with unsuported server versions?

Reply:
Added test for version check .